streaming/src/message/message_bundle.h

```diff
@@ -44,15 +44,29 @@ class StreamingMessageBundleMeta {
 
   StreamingMessageBundleType bundle_type_;
 
+ private:
+  /// To speed up memory copy and serilization, we use memory layout of compiler related
+  /// member variables. It's must be modified if any field is going to be inserted before
+  /// first member property.
+  /// Reference
+  /// :/http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p1113r0.html#2254).
```

Comment:
How big is this optimization?

Reply:
> How big is this optimization?

It will simplify serlization & deserialization of each bundle meta.
In directcall mode, one task produces a bundle message, and this optimization copy a block memory of batch fields(24bytes) not in there times copy (8bytes X 3).

Reply:
Sorry, it's four fields here. [4bytes, 8bytes, 8bytes, 4bytes].